pkg/reaper/resource.go

```diff
@@ -15,8 +15,11 @@ const (
 	DatacenterAvailabilityEach  = "EACH"
 )
 
-func ResourceName(klusterName, dcName string) string {
-	return klusterName + "-" + dcName + "-reaper"
+// DefaultResourceName generates a name for a new Reaper resource that is derived from the Cassandra cluster and DC
+// names.
+func DefaultResourceName(dc *cassdcapi.CassandraDatacenter) string {
+	// FIXME sanitize name
```

Comment:
I'm waiting on #267 for this, but I had to at least align Cassandra, Stargate and Reaper names.